engine/src/w32spec.cpp

```diff
@@ -98,10 +98,59 @@ void MCS_loadfile(MCExecPoint &ep, Boolean binary)
 #ifdef /* MCS_savefile_dsk_w32 */ LEGACY_SYSTEM
 void MCS_savefile(const MCString &fname, MCExecPoint &data, Boolean binary)
 {
-	if (!MCSecureModeCanAccessDisk())
+	char *t_path = NULL;
```

Comment:
Oh, I get the point, I did not see that the change was not put in the right function. Well,`MCS_get_canonical_path` does not exist anymore in develop-7.0, as we updated `ResolvePath` to do so (forgetting Windows in the process).
I suppose that the only thing to do is to remove those lines added in `w32spec.cpp` from the merge, since the changes have been ported to `MCWindowsDesktop::ResolvePath`?